components/datatypes/consentpreferences.schema.json

```diff
@@ -0,0 +1,424 @@
+              "description": "It is okay to present offers via less traditional types of connected devices",
+              "$ref": "#/definitions/marketing-fields"
+            },
+            "xdm:socialMedia": {
+              "title": "Receive Social Media Messages",
+              "description": "It is okay to direct messages to the user via their social media accounts",
+              "$ref": "#/definitions/marketing-fields"
+            }
+          }
+        }
+      }
+    },
+    "consent-metadata": {
+      "properties": {
+        "xdm:version": {
+          "title": "Preferences Version",
+          "description": "Version of the Privacy Preferences Standard",
+          "type": "string",
+          "pattern": "^[0-9]{1,2}\\.[0-9]{1,2}\\.[0-9]{1,4}$",
+          "default": "1.0.0"
+        },
+        "xdm:timestamp": {
+          "title": "Consent Preferences timestamp",
+          "description": "Timestamp of the complete set of user consent preferences",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:source": {
+          "title": "Consent Preferences source",
+          "description": "Source for the complete set of user consent preferences",
+          "$ref": "#/definitions/source"
+        },
+        "xdm:sourceID": {
+          "title": "?? Source ID",
+          "description": "ID for user as defined by consent source",
+          "type": "string",
+          "maxLength": 20
+        },
+        "xdm:userCountry": {
+          "title": "User Country/Region",
+          "description": "ISO 3166 alpha2 country code or ISO 3166-2 country plus state/province code for use in helping to determine how to interpret consent values",
+          "type": "string",
+          "maxLength": 6,
+          "pattern": "[A-Z]{2}(-[A-Z]{1,3}){0,1}"
+        },
+        "xdm:countrySource": {
+          "title": "Country Source",
+          "description": "Method used to determine the user's country",
+          "type": "string",
+          "enum": [
+            "ip",
+            "gps",
+            "user_provided",
+            "website_location",
+            "inferred",
+            "other"
+          ],
+          "meta:enum": {
+            "ip": "IP Address",
+            "gps": "Device GPS",
+            "user_provided": "User Provided",
+            "website_location": "Website location or eTLD",
+            "inferred": "Inferred",
+            "other": "Other"
+          }
+        }
+      }
+    },
+    "consent-preferences": {
+      "properties": {
+        "xdm:preferences": {
+            "$ref": "#/definitions/consent-options"
+        },
+        "xdm:consentMetadata": {
+            "$ref": "#/definitions/consent-metadata"
+        }
+      }
+    }
+  },
+  "allOf":[
+    {
+      "$ref":"https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "#/definitions/consent-preferences"
+    }
+  ],
+  "meta:status":"experimental"
```

Comment:
Change this value to "stable" if you want this to show on Stage and Prod otherwise it will be INT only.